circuits/core/manager.py

```diff
@@ -219,7 +221,7 @@ def __repr__(self):
 
         channel = "/{0:s}".format(str(getattr(self, "channel", "")))
 
-        q = len(self._queue)
+        q = len(self)
```

Comment:
For the same reason we should be explicit here in case someone overrides `__len__` on their components :/ People do silly things :)